ci/jenkins/test.sh

```diff
@@ -839,6 +852,23 @@ function run_conformance {
     kubectl rollout status daemonset/antrea-agent -n kube-system
 
     set +e
+    # Conformance DNS test cases fail in IPv6 Kind cluster because here CoreDNS pods are IPv6 pods 
+    # and it can't reach IPv4 upstream nameserver.
+    # Note: Default CoreDNS ConfigMap uses /etc/resolv.conf to get upstream nameservers listed in resolv.conf file on the host,
+    # the upstream nameservers are accessed by IPv4, when single stack IPv6 pods try to connect IPv4 nameserver it return servfail,
+    # with the new ConfigMap we will forward DNS query to "vmware.com:53" and it will return 'NXDOMAIN' so that client keep searching.
+    # For more detail: https://github.com/kubernetes/kubernetes/issues/94794 
```

Comment:
```suggestion
    # Note: Default CoreDNS ConfigMap uses /etc/resolv.conf to get upstream nameservers listed in resolv.conf file on the host.
    # The upstream nameservers are accessed by IPv4. When single stack IPv6 Pods try to connect IPv4 nameserver, it returns servfail.
    # With the new ConfigMap, it will forward DNS query to "vmware.com:53" for the lab Jenkins and `compute.internal:53` for the cloud Jenkins, and it will return 'NXDOMAIN' so that client keep searching.
    # For more detail: https://github.com/kubernetes/kubernetes/issues/94794 
```
I have a question about the DNS server, why we need compute.internal:53 for the cloud Jenkins?

Reply:
@luolanzone ,
Usually in cloud, internal domain serves as a conditional forwarder on  internal DNS server to forward queries for the domain like compute.internal to the "cloud DNS server". for aws/any other cloud domain severs are located zone wise internally, Usually in region specific domain resolution case it is like region-name.compute.internal etc.

Reply:
Hi, because  in cloud VM search directory in `/etc/resolv.conf ` file is `search us-west-2.compute.internal`  so all the dns query which are not resolved by ipv4 nameserver those query we will handle by  `compute.internal:53` in coredns configMap. 

Example:
`[ERROR] plugin/errors: 2 kubernetes.default.svc.cluster.local.us-west-2.compute.internal. AAAA: dial tcp 172.18.0.1:53: connect: network is unreachable
[ERROR] plugin/errors: 2 kubernetes.default.svc.cluster.local.us-west-2.compute.internal. AAAA: dial udp 172.18.0.1:53: connect: network is unreachable`